openedx/features/survey_report/queries.py

```diff
@@ -0,0 +1,72 @@
+"""
+Queries to get data from database.
+"""
+
+from datetime import datetime, timedelta
+
+from django.contrib.auth.models import User  # lint-amnesty, pylint: disable=imported-auth-user
+from django.db.models import Count, OuterRef, Q, Subquery
+
+from common.djangoapps.util.query import read_replica_or_default
+from common.djangoapps.student.models import CourseEnrollment
+from lms.djangoapps.grades.models import PersistentCourseGrade
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+
+
+def get_unique_courses_offered() -> int:
+    """
+    Get total number of unique course that started before today and have an open date,
+    or have not finished yet, whose number of enrollments is greater than 5.
+    """
+    return CourseOverview.objects.annotate(
+        count=Subquery(
+            CourseEnrollment.objects
+                            .filter(course_id=OuterRef('id'))
+                            .values('course_id')
+                            .annotate(count=Count('course_id'))
+                            .values('count')
+        ))\
+        .filter(count__gt=5)\
```

Comment:
why 5? can we explain that? Also, should this be a constant with a descriptive name?

Reply:
@felipemontoya why 5?, why we sent the sql query to the community with this filter?